geode-core/src/main/java/org/apache/geode/internal/cache/tier/sockets/command/GatewayReceiverCommand.java

```diff
@@ -350,6 +352,7 @@ public void cmdExecute(final Message clientMessage, final ServerConnection serve
               break;
 
             case 1: // Update
+            case GatewaySenderEventImpl.UPDATE_ACTION_NO_GENERATE_CALLBACKS:
               try {
```

Comment:
What happens if the other site is an older version? We need to verify there aren't any versioning issues with the changes to GatewaySenderEventImpl and GatewayReceiverCommand.

Reply:
In case the remote site is in an older version, the batch will be rejected as the operation will not be known.
We will see in the receiving side an "Unknown action type 4 for batch from ..." log message.
This could be stated in the documentation, that in order for this command to work, both sides must be in the version the command was introduced or in a higher one.

Reply:
An alternative would be to detect that the remote side is in an older version and send a regular UPDATE action. Nevertheless, that would make callbacks be invoked so, in my opinion, it is a less safe option.